CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project are documented in this file.
 ### Added
 - Implement `CubicSpline` class (https://github.com/dic-iit/bipedal-locomotion-framework/pull/344)
 - Implement `PWM` control in RobotControl class (https://github.com/dic-iit/bipedal-locomotion-framework/pull/346)
+- Implement `ContactWrenchCone` class in Math component (https://github.com/dic-iit/bipedal-locomotion-framework/pull/352)
+- Implement `skew` function in Math component (https://github.com/dic-iit/bipedal-locomotion-framework/pull/352)
```

Comment:
Just FYI, `skew` exists also in manif for both 2d and 3d cases, 
https://github.com/artivis/manif/blob/7c6b8b2dab6d940204136a5b280db1c243d72c4a/include/manif/impl/eigen.h#L131-L166

Reply:
Yes, I noticed but skew function is defined as "private" implementation in manif. I don't know if it is wise to directly their functions.


Let's ask the others @S-Dafarra and @traversaro

Reply:
Are you sure that it is meant to be private? It is used in the manif examples (see https://github.com/artivis/manif/blob/5d1297b6d8996a5123aea6f1508cd8a029cac8e2/examples/se_2_3_localization.cpp#L368) and the `private` namespace is actually closed before `skew` is defined, see https://github.com/artivis/manif/blob/7c6b8b2dab6d940204136a5b280db1c243d72c4a/include/manif/impl/eigen.h#L129 .

Reply:
Ok! Since the implementation that I wrote is actually  a copy-paste of the manif one I think I can remove the file

Reply:
Removed!